### PR TITLE
Fix return value documentation (1/2)

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Part of #63411. Obvious errors in the return value documentation (typos, wrong type names, wrong keys, wrong indentation).

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
aws_codebuild
aws_codepipeline
aws_kms
aws_kms_info
aws_secret
ec2_vpc_vpn_info
elb_target_group_info
rds
rds_subnet_group
redshift
azure_rm_automationaccount_info
azure_rm_containerregistry_info
azure_rm_devtestlabvirtualmachine_info
azure_rm_dnsrecordset
azure_rm_hdinsightcluster_info
azure_rm_publicipaddress
azure_rm_resource
azure_rm_roleassignment_info
azure_rm_securitygroup_info
azure_rm_subnet_info
azure_rm_virtualnetwork
hcloud_floating_ip
hcloud_network
hcloud_rdns
hcloud_ssh_key
hcloud_volume
hcloud_volume_info
online_server_facts
online_user_facts
online_server_info
online_user_info
os_coe_cluster
os_coe_cluster_template
os_group_info
os_quota
scaleway_image_facts
scaleway_ip_facts
scaleway_organization_facts
scaleway_security_group_facts
scaleway_server_facts
scaleway_snapshot_facts
scaleway_volume_facts
scaleway_image_info
scaleway_ip_info
scaleway_organization_info
scaleway_security_group_info
scaleway_server_info
scaleway_snapshot_info
scaleway_volume_info

### CODE REVIEW

**Comment by LKaemmerling** [comment]
LGTM from my hcloud perspective, Thank you @felixfontein!

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/amazon/ec2_vpc_vpn_info.py [hunk 1]
```diff
@@ -120,26 +120,26 @@
...
-             description: The status of the VPN tunnel.
-             returned: always
-             type: str
-             sample: DOWN
-         status_message:
-             description: If an error occurs, a description of the error.
-             returned: always
-             type: str
-             sample: IPSEC IS DOWN
+           last_status_change:
```
Comment (s-hertel) [inline lib/ansible/modules/cloud/amazon/ec2_vpc_vpn_info.py:123]: Wow, thanks for catching that. There was recently a [new one](https://aws.amazon.com/about-aws/whats-new/2019/08/aws-site-to-site-vpn-now-supports-certificate-authentication/) added that may be returned too:
```
certificate_arn:
    description: The Amazon Resource Name of the virtual private gateway tunnel endpoint certificate.
    returned: when a private certificate is used for authentication
    type: str
    sample: "arn:aws:acm:us-east-1:123456789101:certificate/c544d8ce-20b8-4fff-98b0-example"
```

**Review by s-hertel — APPROVED** [review]

File: lib/ansible/modules/cloud/amazon/ec2_vpc_vpn_info.py (hunk 1 above)
Comment (felixfontein) [reply lib/ansible/modules/cloud/amazon/ec2_vpc_vpn_info.py:123]: As discussed on IRC, this will be added in a follow-up PR.

**Review by resmo — APPROVED** [review]
shipit

**Comment by felixfontein** [comment]
@LKaemmerling @Fred-sun @s-hertel @resmo thanks a lot for reviewing this!
@acozine thanks for merging!

File: lib/ansible/modules/cloud/amazon/ec2_vpc_vpn_info.py (hunk 1 above)
Comment (felixfontein) [reply lib/ansible/modules/cloud/amazon/ec2_vpc_vpn_info.py:123]: I've added it to #63541.

**Pull request merged**